wpeverest/user-registration
# Enhancement - Hidden field

### All Submissions:


### Changes proposed in this Pull Request:

This PR gives the new hidden field in advanced fields which is not visible in the front end.

### Dependency PR : 
[https://github.com/wpeverest/user-registration-advanced-fields/pull/55](url)

### How to test the changes in this Pull Request:

1. Install and Activate the User Registration Advanced Fields addon.
2. Go to add a form or edit a form from all forms -> try to drag the hidden field -> add some default value -> update form or save the form.
3. Check whether the hidden field is visible in the front end or not. Also check hidden field is editable or not from the user list.

### Types of changes:


### Other information:


### Changelog entry

> Enhancement - Hidden field
